src/main/scala/devices/debug/Periphery.scala

```diff
@@ -30,6 +31,11 @@ class DebugIO(implicit val p: Parameters) extends ParameterizedBundle()(p) with
 trait HasPeripheryDebug { this: BaseSubsystem =>
   val debug = LazyModule(new TLDebugModule(pbus.beatBytes))
   pbus.toVariableWidthSlave(Some("debug")){ debug.node }
+
+
+  debug.dmInner.dmInner.sb2tlOpt.foreach { sb2tl  =>
+    fbus.fromPort(Some("debug_sb")){ TLWidthWidget(1) := sb2tl.node }
```

Comment:
can i know the intention behind attaching debug_sb to fbus and not as a coherent master on sbus ?
this becomes an issue when using sba in fesvr/dtm.cc to read to_host which will most likely not be flushed out(by the core) from the cache to memory or is it that in that case you have to use progbuf to read the cache

Reply:
Hey @codelec, see the debug spec at https://github.com/riscv/riscv-debug-spec. This decision was made to reduce complexity of the Debug unit such that it does not need to consider coherence

Reply:
Fesvr uses the program buffer at this time

Reply:
Thank you for the clarification. I was hoping this would give us less invasive polling of the tohost.

Reply:
Yes, we can probably do that in many cases. I don't believe that putting this on the FrontBus excludes it from being coherent, but @hcook may be able to answer that better than I.